src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    get info() {
+      return `name: ${this.name},`
+        + ` chip version: ${this.version}, wheels: ${wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
Keep props together, write methods after.
```suggestion
    coords: {
      x: 0,
      y: 0,
    },
    
    get info() {
      return `name: ${this.name},`
        + ` chip version: ${this.version}, wheels: ${wheels}`;
    },
```